packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -88,6 +87,7 @@ export const Checkbox = compose<ICheckboxType>({
         accessibilityActions: [{ name: 'Toggle', label: checkboxSelectActionLabel }],
         onAccessibilityAction: onAccessibilityAction,
         onKeyUp: onKeyUpSpace,
+        focusable: !state.disabled,
```

Comment:
I think `focusable`, `accessible`, and `accessibiltyRole` is set in Checkbox.settings.tsx. Could you remove them and place them here instead (like you already did with focusable)? 

I wonder why this was working on win32 just fine w/o this change 🤔 either way I think it's a good change to make